test-utils/tests/test-callAsync.js

```diff
@@ -20,6 +20,6 @@ o.spec("callAsync", function() {
 		})
 		timeout = setTimeout(function() {
 			throw new Error("callAsync was called too slow")
-		}, 0)
+		}, 5)
```

Comment:
This is me working around the fact `0` may have it usually running with literally 0ms of a timeout, since it wouldn't be sufficiently nested.